models/VideoModel.js

```diff
@@ -0,0 +1,30 @@
+var VideoModel = Backbone.Model.extend({
+  defaults() {
+    return {
+      id: null,
+      channel: null,
+      title: null,
+      thumbnailUrl: null,
+      description: null,
+      mainPlayer: false
```

Comment:
Thoughts on the way you did the "main video" change. It works and it's not wrong, but I think it may have been simpler to put this attribute (`mainVideo` or `mainPlayer`) on the `appModel` and listen for changes on the `appModel` from the `appView` and just call some `renderMainVideo` function with the `mainVideo` as the model for it.